doc/releases/release-notes-3.4.rst

```diff
@@ -0,0 +1,253 @@
+:orphan:
+
+.. _zephyr_3.4:
+
+Zephyr 3.4.0 (Working Draft)
+############################
+
+We are pleased to announce the release of Zephyr version 3.4.0.
+
+Major enhancements with this release include:
+
+The following sections provide detailed lists of changes by component.
+
+Security Vulnerability Related
+******************************
+
+API Changes
+***********
+
+Changes in this release
+=======================
+
+Removed APIs in this release
+============================
+
+Deprecated in this release
+==========================
+
+Stable API changes in this release
+==================================
+
+New APIs in this release
+========================
+
+Kernel
+******
+
+Architectures
+*************
+
+* ARM
+
+* ARM
+
+* ARM64
+
+* RISC-V
+
+* Xtensa
+
+Bluetooth
+*********
+
+* Audio
+
+* Direction Finding
+
+* Host
+
+* Mesh
+
+* Controller
+
+* HCI Driver
+
+Boards & SoC Support
+********************
+
+* Added support for these SoC series:
+
+* Removed support for these SoC series:
+
+* Made these changes in other SoC series:
+
+* Added support for these ARC boards:
+
+* Added support for these ARM boards:
+
+* Added support for these ARM64 boards:
+
+* Added support for these RISC-V boards:
+
+* Added support for these X86 boards:
+
+* Added support for these Xtensa boards:
+
+* Made these changes for ARC boards:
+
+* Made these changes for ARM boards:
+
+* Made these changes for ARM64 boards:
+
+* Made these changes for RISC-V boards:
+
+* Made these changes for X86 boards:
+
+* Made these changes for Xtensa boards:
+
+* Removed support for these ARC boards:
+
+* Removed support for these ARM boards:
+
+* Removed support for these ARM64 boards:
+
+* Removed support for these RISC-V boards:
+
+* Removed support for these X86 boards:
+
+* Removed support for these Xtensa boards:
+
+* Made these changes in other boards:
+
+* Added support for these following shields:
+
+Build system and infrastructure
+*******************************
+
+Drivers and Sensors
+*******************
+
+* ADC
+
+* Battery-backed RAM
+
+* CAN
+
+* Clock control
+
+* Counter
+
+* Crypto
+
+* DAC
+
+* DFU
+
+* Disk
+
+* Display
+
+* DMA
+
+* EEPROM
+
+* Entropy
+
+* ESPI
+
+* Ethernet
+
+* Flash
+
+* FPGA
+
```

Comment:
Shouldn't Fuel Gauge be added? I guess the section should have probably been there for 3.3 too, to highlight the addition of the SBS driver.

Reply:
Seems it wasn't there because it was at the top as a "newly introduced" section, added.

Reply:
Thanks! And yes, the addition of the new subsystem was definitely there and is luckily hard to miss :)